docs/personalization/content_types.md

```diff
@@ -1,14 +1,24 @@
 # 5. Content Types
 
-In the **advanced** edition of the recommendation engine it is possible to build different recommendation domains. It can be achieved by splitting all the products into different content types. There are several possible use cases for content types.
+In the **advanced** edition of the recommendation engine it is possible to build different recommendation domains. 
```

Comment:
Note for future (out of scope of this PR) - remove mentions of advanced version, I think we don't have the distinction anymore